CRD-DTR/HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json

```diff
@@ -55,11 +55,13 @@
       "linkId": "4",
       "type": "group",
       "text": "Patient diagnoses",
+      "required": true,
```

Comment:
I'm not sure if marking the group as required is necessary.  It seems like this would mean "everything in this group is required", but if that's the case, then the "required" tags on the items aren't needed.  Not sure about this one.